distributions/common/gradle_example.properties

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright 2018 The Data Transfer Project Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Example ~/.gradle/gradle.properties file for secrets when running locally (and not connecting to
+# a cloud project).
+
+data.portability.key.microsoft=foo
+data.portability.secret.microsoft=foo
+data.portability.key.google=bar
+data.portability.secret.google=bar
+data.portability.key.jwt=foo
+data.portability.secret.jwt=bar
+log4j.rootLogger=DEBUG, A1
```

Comment:
Are these properties something new that we added? Don't remember having to put them in ~/gradle.properties

Reply:
You mean the log4j ones right? I think you're right, I think I was just testing that from before so I removed it